server/tserver/src/main/java/org/apache/accumulo/tserver/tablet/Tablet.java

```diff
@@ -603,9 +603,9 @@ private MinorCompactionTask createMinorCompactionTask(long flushId,
             logMessage.append(" tabletMemory.memoryReservedForMinC() "
                 + getTabletMemory().memoryReservedForMinC());
           }
-          if (getTabletMemory() != null && getTabletMemory().getMemTable() != null) {
-            logMessage.append(" tabletMemory.getMemTable().getNumEntries() "
-                + getTabletMemory().getMemTable().getNumEntries());
+          if (mincReason == MinorCompactionReason.SYSTEM) {
+            logMessage.append(" fileCount " + getDatafiles().size());
+            logMessage.append(" fileCountPauseLimit " + pauseLimit);
```

Comment:
Could add another condition so that this information is only appended to the log message when it was the pause limit that caused the tablet to not minor compact.

```suggestion  
          if (mincReason == MinorCompactionReason.SYSTEM && getDatafiles().size() >= pauseLimit) {
            logMessage.append(" fileCount " + getDatafiles().size());
            logMessage.append(" fileCountPauseLimit " + pauseLimit);
```